app/parse.py

```diff
@@ -1,9 +1,41 @@
-from dataclasses import dataclass
+import csv
+from dataclasses import dataclass, astuple, fields
 from urllib.parse import urljoin
-
+from bs4 import Tag, BeautifulSoup
+from selenium import webdriver
+from selenium.common import (
+    ElementNotInteractableException,
+    ElementClickInterceptedException,
+)
+from selenium.webdriver.chrome.webdriver import WebDriver
+from selenium.webdriver.common.by import By
 
 BASE_URL = "https://webscraper.io/"
-HOME_URL = urljoin(BASE_URL, "test-sites/e-commerce/more/")
+urls = (
+    ("home",
+     urljoin(BASE_URL, "test-sites/e-commerce/more")),
+    ("computers",
+     urljoin(BASE_URL, "test-sites/e-commerce/more/computers")),
+    ("laptops",
+     urljoin(BASE_URL, "test-sites/e-commerce/more/computers/laptops")),
+    ("tablets",
+     urljoin(BASE_URL, "test-sites/e-commerce/more/computers/tablets")),
+    ("phones",
+     urljoin(BASE_URL, "test-sites/e-commerce/more/phones")),
+    ("touch",
+     urljoin(BASE_URL, "test-sites/e-commerce/more/phones/touch")),
+)
+
+_driver: WebDriver | None = None
```

Comment:
Using a global variable for the WebDriver instance can lead to issues, especially in larger applications or when running multiple tests. Consider using dependency injection or a context manager to handle the WebDriver instance more safely.